modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PythonExperimentalClientCodegen.java

```diff
@@ -1329,9 +1329,14 @@ private MappedModel getDiscriminatorMappedModel(CodegenDiscriminator disc) {
      *                    ModelName( line 0
      *                        some_property='some_property_example' line 1
      *                    ) line 2
+     * @param includedSchemas schemas already included in this example - for cycle detection.
```

Comment:
Can you further describe includedSchemas here?
includedSchemas are a list of schemas that we have moved through to get here. If the new schemas that we are looking at is in includedSchemas then we have hit a cycle.